riscv/sim.cc

```diff
@@ -36,11 +36,11 @@ sim_t::sim_t(const char* isa, const char* priv, const char* varch,
              std::vector<int> const hartids,
              const debug_module_config_t &dm_config,
              const char *log_path,
-             bool dtb_enabled, const char *dtb_file
+             bool dtb_enabled, const char *dtb_file,
 #ifdef HAVE_BOOST_ASIO
-             , io_service *io_service_ptr_ctor, tcp::acceptor *acceptor_ptr_ctor // option -s
+             io_service *io_service_ptr_ctor, tcp::acceptor *acceptor_ptr_ctor, // option -s
 #endif
-             )
+             FILE *cmd_file_ctor) // neded for command line option --cmd
```

Comment:
spelling of "needed"

Reply:
Instead of naming this `cmd_file_ctor` and doing a `cmd_file = cmd_file_ctor;` later on, you can simply give this the same name `cmd_file` and add `cmd_file(cmd_file)` to the initializer list.  This is what we did for the `mems` argument.  You could also do this for `acceptor_ptr_ctor`.